doc/source/reference/ufuncs.rst

```diff
@@ -429,9 +429,9 @@ Attempting to call these methods on other ufuncs will cause a
 :exc:`ValueError`. The reduce-like methods all take an *axis* keyword
 and a *dtype* keyword, and the arrays must all have dimension >= 1.
 The *axis* keyword specifies the axis of the array over which the reduction
-will take place and may be negative, but must be an integer. The
+will take place and may be negative, in which case it counts from the last to the first axis. *axis* can be an integer, or if in :meth:`ufunc.reduce`, it can also be `None` or `tuple of ints` . The
 *dtype* keyword allows you to manage a very common problem that arises
-when naively using :ref:`{op}.reduce <ufunc.reduce>`. Sometimes you may
+when naively using :meth:`ufunc.reduce`. Sometimes you may
```

Comment:
This was actually correct; e.g., one has `np.add.reduce`, etc. The angled brackets ensured the link would still work.